packages/core/src/components/GlobalActions/GlobalActions.tsx

```diff
@@ -60,62 +51,58 @@ export const HvGlobalActions = (props: HvGlobalActionsProps) => {
     position: positionProp,
     ...others
   } = useDefaultProps("HvGlobalActions", props);
+  const muiTheme = useMuiTheme();
+  const { activeTheme } = useTheme();
+  const { classes, cx, css } = useClasses(classesProp);
+  const isSmDown = useMediaQuery(muiTheme.breakpoints.down("sm"));
+  const isUpMd = useMediaQuery(muiTheme.breakpoints.up("md"));
 
-  const { activeTheme } = useHvTheme();
-
-  const muiTheme = useTheme();
+  const fixedPositionCss =
+    positionProp === "fixed" &&
+    (isUpMd
+      ? {
```

Comment:
An idea: we could move this to the styles file (outside `createClasses`) as an utility function, since this styling block is kinda large. something like

```ts
positionProp === "fixed" && getBreakpointStyles(isUpMd, isSmDown)
```